package.json

```diff
@@ -8,6 +8,12 @@
   "module": "dist/downshift.esm.js",
   "typings": "typings/index.d.ts",
   "scripts": {
+    "start": "nps",
+    "test": "nps test",
+    "prerelease": "npm start validate",
+    "release": "npm publish && npm dist-tag add downshift@$npm_package_version rc",
```

Comment:
I'm also worried that this PR affects how downshift is being released. @kentcdodds do these scripts need to be added? I figured the releasing was already being taken care of by `kcd-scripts`.